src/types/modifications/CCUS.jl

```diff
@@ -3,12 +3,30 @@
 
     CCUS(;file, groupby, co2_scalar=1e5, co2_step_quantity_limit=1e9)
 
-This is a [`Modification`](@ref) that sets up markets for carbon captured by generators.  
+This is a [`Modification`](@ref) that sets up markets for carbon captured by generators.
+
+### Keyword Arguments    
 * `file` - a file to the table containing markets/prices for buying/selling carbon dioxide.  See the `ccus_paths` table below, or [`summarize_table(::Val{:ccus_paths})`](@ref)
 * `groupby` - a `String` indicating how markets are grouped.  I.e. "state".
 * `co2_scalar` - a `Float64` for how much to scale the co2 variables by.  This helps with numerical instability, given that some CO₂ steps can be very large and could bloat the RHS and bounds range of the model.  A good rule of thumb is that this should be no less than `1e4` times smaller than `co2_step_quantity_limit`.
 * `co2_step_quantity_limit` - A `Float64` for the maximum quantity of CO₂ that can be stored in any step.
 
+### Tables Added
+* `<name>_ccus_paths` - contains all the pathways to selling CO₂. The table includes the producing region, the sequstering region and the type of storage, market steps, the quantity available for storage, and the costs of storage and transport.
```

Comment:
I believe the table being created is simply `ccus_paths` (i.e. no name in front of it).